src/task/methods/pyboost/script.py

```diff
@@ -0,0 +1,101 @@
+# This script is based on an IPython notebook:
+# https://github.com/Ambros-M/Single-Cell-Perturbations-2023/blob/main/notebooks/scp-26-py-boost-recommender-system-and-et.ipynb
+
+import numpy as np
+import pandas as pd
+import warnings
+import zipfile
+import sys
+
+warnings.simplefilter('ignore', FutureWarning)
+np.set_printoptions(linewidth=195, edgeitems=3)
+pd.set_option("min_rows", 6)
+
+## VIASH START
+par = dict(
+    de_train = "resources/neurips-2023-data/de_train.parquet",
+    id_map = "resources/neurips-2023-data/id_map.csv",
+    train_obs_zip = "resources/neurips-2023-kaggle/train_obs.csv.zip",
+    predictor_names = ["py_boost"],
+    output = "output.parquet",
+)
+meta = dict(
+    resources_dir = "src/task/methods/ambros_m_judges_third_place"
```

Comment:
Should this be changed to `src/task/methods/pyboost`?

Reply:
Fixed!